apps/api/src/app/workflows-v2/generate-preview.e2e.ts

```diff
@@ -354,13 +354,19 @@ function buildChatControlValuesPayload() {
     body: 'Hello, World! {{subscriber.firstName}}',
   };
 }
+function buildDigestControlValuesPayload() {
+  return {
+    cron: '0 3 * * 0',
```

Comment:
We have a [`CronExpression` util](https://github.com/novuhq/novu/blob/next/packages/shared/src/types/cron/cron.types.ts#L5) in shared we can used here for better readability